manticore/__main__.py

```diff
@@ -103,75 +103,10 @@ def ethereum_cli(args):
-    #         print "Transaction:"
-    #         print "\tsort %s" % tx.sort  # Last instruction or type? TBD
-    #         print "\tcaller 0x%x" % state.solve_one(
-    #             tx.caller)  # The caller as by the yellow paper
-    #         print "\taddress 0x%x" % state.solve_one(
-    #             tx.address)  # The address as by the yellow paper
-    #         print "\tvalue: %d" % state.solve_one(
-    #             tx.value)  # The value as by the yellow paper
-    #         print "\tcalldata: %r" % state.solve_one(tx.data)
-    #         print "\tresult: %s" % tx.result  # The result if any RETURN or REVERT
-    #         print "\treturn_data: %r" % state.solve_one(
-    #             tx.return_data)  # The returned data if RETURN or REVERT
-    #
-    #         if tx.sort == 'Call':
-    #             metadata = seth.get_metadata(tx.address)
-    #             if metadata is not None:
-    #                 function_id = tx.data[:4]  # hope there is enough data
-    #                 function_id = state.solve_one(function_id).encode('hex')
-    #                 signature = metadata.get_func_signature(function_id)
-    #                 print "\tparsed calldata", ABI.parse(signature,
-    #                                                      tx.data)  # func_id, *function arguments
-    #                 if tx.result == 'RETURN':
-    #                     ret_types = metadata.get_func_return_types(function_id)
-    #                     print '\tparsed return_data', ABI.parse(ret_types,
-    #                                                             tx.return_data)  # function return
-    #
-    #     # the logs
-    #     for log_item in blockchain.logs:
-    #         print "log address", log_item.address
-    #         print "memlog", log_item.memlog
-    #         for topic in log_item.topics:
-    #             print "topic", topic
-    #
-    #     for address in blockchain.deleted_addresses:
-    #         print "deleted address", address  # selfdestructed address
-    #
-    #     # accounts alive in this state
-    #     for address in blockchain.contract_accounts:
-    #         code = blockchain.get_code(address)
-    #         balance = blockchain.get_balance(address)
-    #         trace = set((offset for address_i, offset in state.context['seth.trace'] if
-    #                      address == address_i))
-    #         print calculate_coverage(code, trace)  # coverage % for address in this state
-    #
-    #         # All accounts ever created by the script
-    #         # (may not all be alife in all states)
-    #         # (accounts created by contract code are not in this list )
-    # print "[+] Global coverage:"
-    # for address in seth.contract_accounts:
-    #     print address, seth.global_coverage(
-    #         address)  # coverage % for address in this state
+
+    m.finalize()
+    print "[+] Look for results in %s"% seth.workspace
```

Comment:
yikes, seth.workspace

Reply:
What is the correct way to send ppl to the workspace folder?

Reply:
`m.workspace` ;)